doc/modules/cross_validation.rst

```diff
@@ -624,6 +624,9 @@ model is flexible enough to learn from highly person specific features it
 could fail to generalize to new subjects. :class:`GroupKFold` makes it possible
 to detect this kind of overfitting situations.
 
+:class:`GroupKFold` is the same as :class:`LeaveOneGroupOut` in the case where
+`n_splits` is equal to the number of groups.
```

Comment:
I'm thinking this information is better in the `LeaveOneGroupOut` section. As a reader learning about just `GroupKFold`, I think this is too much information. But if I was reading about `LeaveOneGroupOut`, it is nice to know that `LeaveOneGroupOut` is a special case of `GroupKFold`.